packages/clerk-js/src/ui/common/Billing.tsx

```diff
@@ -0,0 +1,204 @@
+import type { BillingPlanResource } from '@clerk/types';
+import React from 'react';
+
+import { Badge, Box, Button, Col, descriptors, Flex, Grid, Icon, localizationKeys, Text } from '../customizables';
+import { Header, IconButton } from '../elements';
+import { ArrowLeftIcon, Check } from '../icons';
+
+export type OrganizationPlanCardProps = Pick<BillingPlanResource, 'name' | 'features' | 'priceInCents'> & {
+  isCurrentPlan: boolean;
+  planKey: string;
+  changePlanAction: React.ReactNode;
+};
+
+export const OrganizationPlanCard = (params: OrganizationPlanCardProps) => {
+  return (
+    <Col
+      elementDescriptor={descriptors.billingPlanCard}
+      elementId={descriptors.billingPlanCard.setId(params.planKey)}
```

Comment:
@anagstef @octoper Do you have any objection on using dynamic ID for the element descriptor?

Reply:
@AlexNti unfortunately, elementId is used to generate Public API classnames in the form of `elementDescriptor__elemendId`. So, this is considered Public API that any user can target. :(

Reply:
After discussing with @AlexNti , these plan keys will be set by the developer, so this is totally ok!